deploy/dev.yml

```diff
@@ -44,7 +44,7 @@ services:
             test: curl --fail http://localhost:4002/_health || exit 1
             interval: 10s
             retries: 5
-            start_period: 60s
+            start_period: 180s
```

Comment:
📓 I increased this since starting docker compose from scratch takes longer than a minute.